### DISCUSSION TITLE
Do you want desktop UI app?

### DISCUSSION THREAD

**Discussion by TheLastGimbus** [TheLastGimbus/GooglePhotosTakeoutHelper#190] [General]
Hey everyone!

Since this is all Dart now, i've been thinking of making this a whole big desktop app. Do you think this would be cool for *you* / other new users?

Pros:
- i could release it to microsoft store, and installation may be easier then - the windoza defender also won't scream anymore
- it may be a bit easier for newbies to nicely see what's happening, in colorful progress bars etc
- pure standalone cli will still be there
- develping it wouldn't be that much of work

Cons:
- ...not much, but still **some** work for me - coding stuff that **works** is never easy
- bloat?

Main question is: was using cli-ish version was anyhow hard for any of you? Did you find yourself confused when windoza defender was screaming? etc

**Comment by istituzionealsole** [comment]
That would be nice, I think you'd reach far more people with a desktop app, the terminal scares a lot of people.

**Comment by YousefMenesy** [comment]
That would be awesome! i habe been struggling till now to make it work, i am not a programmer and have not been able to figure it out, it would be great is someone could get in touch with me to get it going, i have more than 70 gb of pictures that i need to get organised to my Synology :D

**Reply by TheLastGimbus** [reply]
How are you "not able to figure out" - if you broadly describe what is going on, someone might help you. Or, you may realize what is wrong while describing :DDD

**Comment by ggortan** [comment]
I think there is no need for desktop UI since is SO SIMPLE to use, but for beginner users desktop UI may help :). Thanks so much for your hard work and for this awesome tool, my 30k plus photos are thanked aswell since they are now out of google photos with correct dates 👍
